esp-box/esp-box.c

```diff
@@ -495,7 +495,6 @@ bool bsp_button_get(const bsp_button_t btn)
 static uint8_t bsp_get_main_button(void *param)
 {
     assert(tp);
-    ESP_ERROR_CHECK(esp_lcd_touch_read_data(tp));
```

Comment:
Are you sure, that it will be working without this line?

Reply:
> Are you sure, that it will be working without this line?

Yes, it worked.
Because lvgl will call `esp_lcd_touch_read_data`, it'll get the btn event of the TP.

Reply:
Ok, you are right. I think, that it is OK now.